### ISSUE TITLE
Very Bad FPS on relatively High-End PC

### ISSUE THREAD

**Issue by LukeAWakefield** [EnigmaticaModpacks/Enigmatica8#71]
### Modpack Version

1.0.4

### Describe your issue.

When in game, I get very bad FPS (as low as 10fps and generally no higher than 30-40 at any time). 

PC Spec:
- CPU: 16x 11th Gen Intel(R) Core(TM) i9-11900k @ 3.50GHz
- Display: 2560x1440
- NVIGIA GeForce RTX 3080/PCie/SSE2
- 16GB RAM (2x8gb) DDR4 3200mhz
- GPU Drivers up to Date (GeForce Game Ready Driver Version 511.79)

Java Version 17.0.1 64Bit
Allocated Memory: 7040MB

I've tried other 1.18.1 Modpacks and had no issues with FPS, such as All the Mods 7 which has an even bigger modlist and it runs comfortably above 100+FPS.

![Enigmatica 8 Screenshot](https://user-images.githubusercontent.com/94404846/154802855-be40b178-b725-4bcf-9881-536265c0ccb3.jpg)

### Have you modified the modpack?

No

### Did the issue happen in singleplayer or on a server?

Singleplayer

**Comment by LukeAWakefield** [comment]
Oh I should also say that in Task Manager, none of my CPU, GPU, Memory or Disk are peaking whilst playing. Around 20-30% Total CPU Usage, 75% Total Memory and 20% Total GPU Usage.

**Comment by Darkere** [comment]
Any chance you have vsync on?

**Comment by LukeAWakefield** [comment]
Yeah tried V-Sync off. Even tried the video settings in the attached screenshot and the frames jump between 40-70fps but there are still horrible microstutters. (And a render distance of 2 chunks is not exactly playable lol)

![2022-02-19_13 51 52](https://user-images.githubusercontent.com/94404846/154803714-02f449fe-deef-4b8b-a96d-030e47e421b7.jpeg)
.

**Comment by NillerMedDild** [comment]
Would you mind removing FPS Reducer and see if the issue persists? :)

**Comment by LukeAWakefield** [comment]
That's the one! Runs much, much better now. Getting 100+FPS and no stutters.

Thank you both for your help. And so quickly too!

**Comment by NillerMedDild** [comment]
My pleasure, now to find out why it's reducing FPS while you're active 🤔

**Comment by Darkere** [comment]
FPS reducer reduces the fps to 1 frame per second or even less IIRC? I don't think it was the actual cause.

**Comment by NillerMedDild** [comment]
It's highly configurable, it can't be used for general throttling too iirc.

**Comment by Darkere** [comment]
```
By default, if there is no operation for 5 minutes the frame rate will drop to 10.

Pressing [PAUSE] key reduces FPS manually. (new feature in v1.16)```

ok then...
maybe we can unbind that to avoid people accidently pressing the button?

**Comment by NillerMedDild** [comment]
I thought I already did that 😅
